ui/app/styles/components/popup-menu.scss

```diff
@@ -55,7 +55,8 @@
     &:hover {
       background-color: $ui-gray-050;
     }
-    div {
```

Comment:
the misalignment had been bothering me, so decided to fix

# before/after
<img width="708" alt="Screenshot 2024-09-27 at 12 05 32 PM" src="https://github.com/user-attachments/assets/b24dcf58-24ee-491a-bc89-eb4600e53064">